.github/workflows/mocha.yml

```diff
@@ -8,37 +8,8 @@ name: Tests
       - reopened
 
 jobs:
-  prepare-commit-msg:
-    name: Retrieve head commit message
-    runs-on: ubuntu-latest
-    # Run 'pull-request' event only on external PRs from forked repos.
-    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
I added this line to prevent identical tests running twice. Once by the `push` event and a second time by the `pull-request` event. This only happened when a maintainer pushes to Mocha's repo.
The tests again will run twice without any additional value.

Reply:
Oops. I remember what we discussed about that. 
I will fix this PR to remove skip ci and keep running twice.